assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -115,13 +113,13 @@ che.workspace.sidecar.default_memory_request_mb=64
 
 # CPU limit default for each sidecar that has no CPU settings in the {prod-short} plug-in configuration.
 # Specify either in floating point cores number, for example, `0.125`,
-# or using the Kubernetes format, integer millicores, for example, `125m`.
+# or using the {orch-name} format, integer millicores, for example, `125m`.
 # Value less or equal to 0 is interpreted as disabling the limit.
 che.workspace.sidecar.default_cpu_limit_cores=-1
 
 # CPU request default for each sidecar that has no CPU settings in the {prod-short} plug-in configuration.
 # Specify either in floating point cores number, for example, `0.125`,
-# or using the Kubernetes format, integer millicores, for example, `125m`.
+# or using the {orch-name} format, integer millicores, for example, `125m`.
```

Comment:
later in this PR you replace Kubernetes with {kubernetes} instead of {orch-name}. Is this a typo, or are you intentionally replacing SOME occurrences with one thing and others with a different variable? eg., look at line 342 https://github.com/eclipse/che/pull/19604/files#diff-deca31b9cca47d5b0356dd75fc0fd0c8cb980f3aa029da27b475bdbb37353eb0R342

Reply:
That's intentional, as long as I didn't make a mistake on what is for "Kubernetes or OpenShift" and what is for "Kubernetes only".

The substitution scheme is different, and the attribute is good for disambiguation for things that are specific to Kubernetes *only*:

`{orch-name}`:  `Kubernetes` in upstream, `OpenShift` in downstream
{kubernetes}: `Kubernetes` everywhere

We have also:
`{platforms-name}`:  `Kubernetes or OpenShift` in upstream, `OpenShift` in downstream

Reply:
@themr0c are you sure `OpenShift format` is a good replacement here. I have some doubts?

Reply:
I am not sure. We default to avoiding any occurrence of the `Kubernetes` word in downstream, but when there is some evidence that we should keep `Kubernetes`, we have the `{kubernetes}` attribute. I see @sparkoo has the same opinion.